pyproject.toml

```diff
@@ -74,6 +74,7 @@ pydantic = { version = "<2.0.0", optional = true }
 requests = { version = "^2.31.0", optional = true }
 starlette = { version = "^0.31.0", optional = true }
 uvicorn = { version = "^0.23.0", extras = ["standard"], optional = true }
+pytest-watcher = "^0.4.1"
```

Comment:
This should be listed under dev dependencies (and use `==` like other dev deps)

Reply:
Makes sense. Will adjust.